diff --git a/modules/wrc_core/xwr_subsystem.vhd b/modules/wrc_core/xwr_subsystem.vhd
index 8a3c7aa..b3f5598 100644
--- a/modules/wrc_core/xwr_subsystem.vhd
+++ b/modules/wrc_core/xwr_subsystem.vhd
@@ -1114,7 +1114,7 @@ begin
       mux_snk_i(0) => nic_snk_in,
       mux_snk_i(1) => wrf_snk_i,
       mux_class_i(0) => x"0f",
-      mux_class_i(1) => x"f0");
+      mux_class_i(1) => x"ff");
 
   -----------------------------------------------------------------------------
   -- External Tx Timestamping I/F
